conf/services/claim-child-benefit.cy.yml

```diff
@@ -0,0 +1,18 @@
+serviceName: Hawlio Budd-dal Plant
+serviceHeaderName: Hawlio Budd-dal Plant
+serviceDescription: Defnyddiwch y ffurflen hon i hawlio Budd-dal Plant os ydych chi neu’ch partner yn gyfrifol am blentyn sydd o dan 16 oed, neu o dan 20 oed os yw’n parhau ag addysg neu hyfforddiant cymeradwy.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /fill-online/claim-child-benefit
+contactFrontendServiceId: claim-child-benefit
+businessArea: Customer Services Group (CSG)
+ddc: DDC Newcastle
+complianceStatus: partial
+statementVisibility: public
+statementCreatedDate: 2022-10-04
+statementLastUpdatedDate: 2023-11-23
+accessibilityProblems:
+- Defnyddiwch y ffurflen hon i hawlio Budd-dal Plant os ydych chi neu’ch partner yn gyfrifol am blentyn sydd o dan 16 oed, neu o dan 20 oed os yw’n parhau ag addysg neu hyfforddiant cymeradwy.
```

Comment:
Looks like there was a copy & paste error for this line; since you're off today, I'll update this.